frontend/public/locales/en/translation.json

```diff
@@ -2824,9 +2830,11 @@
   "The value must be a fully-qualified host name or IP address. Do not include the '{{scheme}}://' URL scheme.": "The value must be a fully-qualified host name or IP address. Do not include the '{{scheme}}://' URL scheme.",
   "The version or versions for the deployable. You can use a range of versions in the form >1.0, or <3.0.": "The version or versions for the deployable. You can use a range of versions in the form >1.0, or <3.0.",
   "There are no Argo applications created. Check the following resources and make sure they are configured properly: applicationset placement, gitopscluster, gitopscluster placement, managedclusterset. Also make sure the ApplicationSet feature is enabled if GitOps is deployed to a namespace other than openshift-gitops.": "There are no Argo applications created. Check the following resources and make sure they are configured properly: applicationset placement, gitopscluster, gitopscluster placement, managedclusterset. Also make sure the ApplicationSet feature is enabled if GitOps is deployed to a namespace other than openshift-gitops.",
+  "There are no policies": "There are no policies",
```

Comment:
Can we add a period here @yiraeChristineKim

Reply:
what about `You don't have any policies` should we add a period too?

Reply:
Yea I recommend adding it there too